webdriver/tests/bidi/browsing_context/context_created/context_created.py

```diff
@@ -134,19 +134,21 @@ async def on_event(method, data):
     children_info = root_info["children"]
     assert len(children_info) == 2
 
+    # Note: Live `browsingContext.contextCreated` events are always created with "about:blank":
+    # https://github.com/w3c/webdriver-bidi/issues/220#issuecomment-1145785349
```

Comment:
I don't think that we have to explicitly refer to the GitHub issue here. I would be fine with the following:

```suggestion
    # Note: Live events are always created with "about:blank":
```